### DISCUSSION TITLE
Git 활용 진짜 빡세네^____^

### DISCUSSION THREAD

**Discussion by tastekim** [tastekim/WeAllLie-BE#13] [Show and tell]
별의 별 기능이 많아서 마치 블로거가 되어가는 기분이지만,,,
그래도 어차피 git을 버릴 수 없다면 노션을 버립시다 여러분,,,!!!🔥

**Comment by ghost** [comment]
ㅋㅋㅋㅋㅋ 깃 열심히 활용하겠습니다!
